x-pack/plugins/ingest_manager/server/services/epm/elasticsearch/template/install.ts

```diff
@@ -124,8 +123,15 @@ const installPreBuiltComponentTemplates = async (
     // This uses the catch-all endpoint 'transport.request' because there is no
     // convenience endpoint for component templates yet.
     // See src/core/server/elasticsearch/api_types.ts for available endpoints.
-    await callCluster('transport.request', callClusterParams);
+    return await callCluster('transport.request', callClusterParams);
```

Comment:
nitpick the await is probably useless here 
```suggestion
return callCluster('transport.request', callClusterParams);
```